src/plugins/LADTable/LADTableCompositionPolicy.js

```diff
@@ -0,0 +1,37 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2020, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define([], function () {
+    return function LADTableCompositionPolicy(openmct) {
```

Comment:
Since we use the new operator to instantiate this (which is the right approach 👍 ) let's make this bad boy a class.

Reply:
Done!

Reply:
OK, on reflection this was nonsense. I had forgotten that composition policies are functions, not objects. This is a gallant effort to comply with completely nonsensical advice. I was thrown by the use of the `new` operator for a function. We don't need to instantiate anything here, there's no state to store.

This should just return a function, it should not be a class. I was confused.